apprentice_teacher.rb

```diff
@@ -1,23 +1,22 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'refactored_code'
+#Getting error: /Users/apprentice/Desktop/phase-1-checkpoint-challenge-3-teacher-refactoring/apprentice_teacher.rb:1:in
+#`require_relative': /Users/apprentice/Desktop/phase-1-checkpoint-challenge-3-teacher-refactoring/refactored_code.rb:38:
+#syntax error, unexpected keyword_end (SyntaxError)
+
+class ApprenticeTeacher < Person
+  attr_reader :salary, :target_raise
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
-    @phase = 3
   end
 
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
+  include Fivable
+  include Phasable
+  include Raisable
+  include Recievable
 
+  #this can be re-factored more, if we can alter the responses inside the actual class
   def teach_stuff
```

Comment:
Totally agree w your comment - how can this method be DRY'd up for the apprentice and teacher?